.idea/.name

```diff
@@ -0,0 +1 @@
+react-starter-kit-goose
```

Comment:
you should understand the process and be able to subordinate technologies, this is a big deal. So answer "I do not know" is not satisfied :)